### PR TITLE
add in some principles

### PR DESCRIPTION
**Pull request by andymckay**
... also @mconca might want to comment

### CODE REVIEW

File: source/new.rst [hunk 1]
```diff
@@ -5,78 +5,107 @@ New API Guidelines
...
 
 Compared to legacy Firefox add-ons there is a significant sandbox around WebExtension APIs that is very restrictive. Access to priviledged APIs within Firefox, preferences, arbitrary File System access, socket access and so on are restricted (at the time of writing this documentation). APIs that break out of this sandbox will need very clear justification for why.
 
-Triaging APIs
--------------
+General principles
+------------------
 
-If you file a bug asking for an API it will go through the bug triage process. Since this is useful to how APIs land, here's what currently happens in bug triages:
+As we've been triaging bugs, we've come up some general principles and guidelines. These are liable to change and adapt as we go along.
```
Comment (mconca) [inline source/new.rst]: ...come up WITH some general...

File: source/new.rst [hunk 2]
```diff
@@ -5,78 +5,107 @@ New API Guidelines
...
 
-Weekly new bug triage
-~~~~~~~~~~~~~~~~~~~~~
+Users over Developers
++++++++++++++++++++++
 
-We look at all new bugs in an attempt to spot serious bugs, regressions or other issues. We try to give each bug a priority and mark as *[triaged]* in the whiteboard. The point here is to do an initial triage and catch critical things. We also label bugs that might be good for contributors or need thinking about. The latter are marked *[design-decision-needed]*, but its important to point out that straightforward change or obvious bugs just go through.
+One example was the request to prevent users from dragging `browserAction buttons into the overflow menu <https://bugzilla.mozilla.org/show_bug.cgi?id=1314070>`_, tab menu or customize menu. That's because its the users choice where they place their buttons, *not* the developers.
```
Comment (mconca) [inline source/new.rst]: This bug was denied because...

**Review by mconca — APPROVED** [review]
Very nice, Andy. No major issues.

File: source/new.rst [hunk 3]
```diff
@@ -5,78 +5,107 @@ New API Guidelines
...
+
+Triaging APIs
+-------------
+
+If you file a bug asking for an API it will go through the bug triage process. Since this is useful to how APIs land, here's what currently happens in bug triages:
+
+Weekly new bug triage
++++++++++++++++++++++
+
+We look at all new bugs in an attempt to spot serious bugs, regressions or other issues. We try to give each bug a priority and mark as *[triaged]* in the whiteboard. The point here is to do an initial triage and catch critical things. We also label bugs that might be good for contributors or need thinking about. The latter are marked *[design-decision-needed]*, but its important to point out that straightforward change or obvious bugs just go through.
```
Comment (aswan) [inline source/new.rst]: this actually isn't the current practice any more...

**Review by aswan — APPROVED** [review]
a few line items comments below but ship it!

File: source/new.rst [hunk 4]
```diff
@@ -5,78 +5,107 @@ New API Guidelines
...
 
-We look at all new bugs in an attempt to spot serious bugs, regressions or other issues. We try to give each bug a priority and mark as *[triaged]* in the whiteboard. The point here is to do an initial triage and catch critical things. We also label bugs that might be good for contributors or need thinking about. The latter are marked *[design-decision-needed]*, but its important to point out that straightforward change or obvious bugs just go through.
+One example was the request to prevent users from dragging `browserAction buttons into the overflow menu <https://bugzilla.mozilla.org/show_bug.cgi?id=1314070>`_, tab menu or customize menu. That's because its the users choice where they place their buttons, *not* the developers.
 
+Users understanding UI changes
+++++++++++++++++++++++++++++++
 
-Bi-weekly community meeting
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Where possible we should try to make sure that changes to Firefox are due to extensions. If a UI change is going to occur, it should occur on install of the extension or in a user action. Where possible that UI that is added should have some clear trace back to the extension that installed it, maybe through icons or text.
```
Comment (aswan) [inline source/new.rst]: I don't understand this, did you mean something like "...make sure that changes to Firefox made by extensions are clear to the user"?

**Comment by andymckay** [comment]
Thanks both, cleaned up based on feedback.

**Pull request merged**